liquid-vector/liquid-vector.cabal

```diff
@@ -22,8 +22,8 @@ library
                       Data.Vector.Internal.Check
                       Data.Vector.Unboxed.Mutable
   hs-source-dirs:     src
-  build-depends:      liquid-base          >= 0.8.10.1
-                    , vector               >= 0.10
+  build-depends:      liquid-base          < 5
```

Comment:
Should be `< 4.15` because that's what `vector` has.

Reply:
Yes, this (and all other comments) are fair enough. As said, I don't have a strong intuition whether putting too-stringent constraints might prevent people from actually experimenting with `liquidhaskell`, i.e. a wanted to avoid restricting ourselves into "this build only with this precise constellation of packages / LTS version".

I guess I will amend the PR with your suggestions and we'll see what happens once we release 😸

Reply:
Well, I think that if you want to make these more liberal, that would make sense if there was a Stackage snapshot that has less recent versions, but then I'd probably still make the LH version number equal to the lowest "supported" one. And I think it's fair to be flexible in the version bounds according to PVP, but I would not go further. It's easier to make revisions and relax them later than have breaking builds due to missing them.